ev3dev2/motor.py

```diff
@@ -2070,7 +2054,8 @@ def follow_line(self,
                 tank.stop()
                 raise
         """
-        assert self._cs, "ColorSensor must be defined"
+        if not self._cs:
+            raise DeviceNotDefined("The 'cs' variable must be defined with a ColorSensor. Example: tank.cs = ColorSensor()")
```

Comment:
Changing from an `assert` to raising `DeviceNotDefined` is a breaking change but the odds of someone getting burned by this are pretty low.  They would have to be using `MoveTank.cs` without one defined but catching the assert to ignore the error.